health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/template_components/RegistrationComponents.js

```
@@ -28,25 +42,197 @@ import { registerComponent } from "./RegistrationRegistry";
+    width: "1px",
+    height: "16px",
+    backgroundColor: "#aaa",
+  },
+};
+
+export const getTemplateRenderer = (templateName) => {
+  switch (templateName) {
+    case "BeneficiaryAcknowledgement":
+    case "HouseholdAcknowledgement":
+      return responsePanelComponent;
+    case "HouseholdOverview":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)